views/home_massAddTable.mako

```diff
@@ -25,8 +25,6 @@
         indexer = 0
         if curDir['existing_info'][0]:
             indexer = curDir['existing_info'][2]
-        elif app.INDEXER_DEFAULT > 0:
```

Comment:
I changed the way this page works a little last time, when I made sure it supported multiple indexers. It doesn't make sense to use the default indexer here.